### PR TITLE
feat: Added import-service

### PR DESCRIPTION
**Pull request by lexBogos**
- File serverless.yml contains configuration for importProductsFile function +
- The importProductsFile lambda function returns a correct response which can be used to upload a file into the S3 bucket +
- Frontend application is integrated with importProductsFile lambda +

https://d1sv461iq579q4.cloudfront.net/admin/products

### CODE REVIEW

File: product-service/src/import-service/serverless.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+service: product-service
```
Comment (IvanPresmytsky) [inline product-service/src/import-service/serverless.yml:1]: Seems you forgot to remove this file

File: product-service/src/import-service/handler.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { importProductsFile } from './src/handlers/get-product-by-id.js';
+
+
+
+export {
+  importProductsFile,
+}
```
Comment (IvanPresmytsky) [inline product-service/src/import-service/handler.js:7]: Seems you forgot to remove this file

File: services/src/import-service/handlers/importProductsFile.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import  AWS  from 'aws-sdk';
+const BUCKET = 'task-5-lex';
```
Comment (IvanPresmytsky) [inline services/src/import-service/handlers/importProductsFile.js:2]: It's better to move it to '.env' file

**Review by IvanPresmytsky — APPROVED** [review]
Good job! But seems you committed some redundant files.

File: product-service/src/import-service/handler.js (hunk 2 above)
Comment (lexBogos) [reply product-service/src/import-service/handler.js]: Nope, that is should be here.

File: product-service/src/import-service/serverless.yml (hunk 1 above)
Comment (lexBogos) [reply product-service/src/import-service/serverless.yml]: I forgot to push last changes))) This name fixed.

File: services/src/import-service/handlers/importProductsFile.js (hunk 3 above)
Comment (lexBogos) [reply services/src/import-service/handlers/importProductsFile.js:2]: Agree!

File: product-service/src/import-service/handler.js (hunk 2 above)
Comment (lexBogos) [reply product-service/src/import-service/handler.js]: > Good job! But seems you committed some redundant files.
> Also please check CORS, I did not succeed to load a file from your FE

Link should be worked from yesterday, code in AWS is not changed from Sunday, tested 5 min ago the publishing to s3 working as expected, please recheck.

Comment (lexBogos) [reply product-service/src/import-service/handler.js]: This is final solution, I have no time to fix((( please take mark for the current scope.

**Pull request merged**